scripts/populate_file_web_server.py

```diff
@@ -72,40 +74,27 @@ def __str__(self):
 
 # Helper functions
 
-def info_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_INFO]:
-        print(msg)
-
-def info_print_in_place(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_INFO]:
-        print(msg, end='\r')
-
-def verbose_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_VERBOSE]:
-        print(msg)
+def print_msg(print_level, msg, print_in_place):
+    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[print_level]:
+        if True == print_in_place:
+            print(msg, end='\r')
+        else:
+            print(msg)
 
-def warn_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_WARN]:
-        print(msg)
-
-def error_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_ERROR]:
-        print(msg)
-        
 def create_dir_if_not_exist(dir):
     if not os.path.exists(dir):
         try:
             os.mkdir(dir)
         except:
-            warn_print("Cannot create directory " + dir)
+            print_msg(PRINT_LEVEL_WARN, "Cannot create directory " + dir, False)
```

Comment:
Please remove False after setting the default value. Applies to other changes below

Reply:
Done